srcpkgs/typecatcher/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'typecatcher'
+pkgname=typecatcher
+version=0.4
+revision=1
+build_style=python3-module
+hostmakedepends="intltool python3-distutils-extra python3-gobject gobject-introspection"
+makedepends="gtk+3-devel webkit2gtk-devel "
+depends="gtk+3 python3 python3-gobject webkit2gtk yelp"
```

Comment:
You can drop `gtk+3` and `webkit2gtk` here, they'll be autodetected.

Reply:
@abenson ok wait..